### PR TITLE
[OKU-19] Unit tests - Reviews

### PR DESCRIPTION
**Pull request by enricodelarosa**

### CODE REVIEW

**Comment by enricodelarosa** [comment]
Test Result Screenshot
image

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "okunokumo.be",
     "version": "1.0.0",
-    "description": "",
+    "description": "All 2 test suites and 10 tests passed with no errors, achieving 71.61% test coverage. Notable successes include passing all authentication tests and successfully adding and approving reviews.",
```
Comment (idiglove) [inline package.json]: Nah you dont have to do this
I meant on description of your PR in Github haha

Comment (enricodelarosa) [reply package.json]: Oh alright my bad haha removinggg

**Review by idiglove — APPROVED** [review]
all good 🏄🏽

**Pull request merged**